index.html

```diff
@@ -184,92 +197,147 @@ <h2 class="work_title">Multi-Post Stories</h2>
+        </div>
+        <div class="about-framework">
+          <ul class="about_ul">
+            <h3 class="sub_title">
+              Languages
+              <img
+                class="down_arrow_icon"
+                src="./images/down-arrow-icon.png"
+                alt="down"
+              />
+            </h3>
+            <div class="frame_work_items">
+              <li class="ul_item">
+                <img src="./images/js-icon.png" alt="Javascript" /><span
+                  >Javascript</span
+                >
+              </li>
+              <li class="ul_item">
+                <img src="./images/html-icon.png" alt="HTML" /><span>HTML</span>
+              </li>
+              <li class="ul_item">
+                <img src="./images/css-icon.png" alt="CSS" /><span>CSS</span>
+              </li>
+            </div>
+          </ul>
+          <ul class="about_ul">
+            <h3 class="sub_title">
+              Frameworks
+              <img
+                class="arrow_icon"
+                src="./images/right-arrow-icon.png"
+                alt="down"
+              />
+            </h3>
+          </ul>
+          <ul class="about_ul">
+            <h3 class="sub_title">
+              Skills
+              <img
+                class="arrow_icon"
+                src="./images/right-arrow-icon.png"
+                alt="down"
+              />
+            </h3>
+          </ul>
+        </div>
+      </section>
+
+      <!--contact form section-->
+      <section class="contact_form" id="contact">
+        <div class="contact_container">
+          <h2 class="title">Contact me</h2>
+          <p class="contact_form_des">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <form
+            class="form_container"
+            action="https://formspree.io/f/mnqroaaw"
+            method="POST"
+          >
+            <fieldset class="form_inputs">
+              <input
+                class="form_input"
+                name="name"
+                type="text"
+                placeholder="Your Name"
+                maxlength="30"
+                required
+              />
+              <input
+                class="form_input"
+                name="email"
+                type="email"
+                placeholder="example@me.com"
+                required
+              />
+              <textarea
+                class="form_text_area"
+                name="message"
+                placeholder="Write your message here"
+                maxlength="500"
+                required
+              ></textarea>
+              <button type="submit" class="btn_submit">Get It Touch</button>
+            </fieldset>
+          </form>
+        </div>
+        <div class="slide">
+          <img src="./images/Shape.png" alt="">
         </div>
```

Comment:
- [ ] Good job on reducing the size of the form 👏 however now you have an additional line on the right of the form which doesn't exist on the Figma design, kindly remove this line and center the content of the contact section 👍

## Your design:
![image](https://user-images.githubusercontent.com/50721479/199986533-f866a22c-b40d-42a4-bddb-f1e32242fd8c.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/199986577-ebb67445-33a3-4b60-b912-3a8f4a281f16.png)